Specs/DateTime/English/DateTimeModel.json

```diff
@@ -11745,5 +11745,101 @@
         }
       }
     ]
+  },
+  {
+    "Input": "2017 -q1",
```

Comment:
This is incorrect, the pattern does not include whitespace. It should be like "2017-q1".
Also, can you add full sentences in the specs instead of just the entity? For example: "The sales went up during 2017-q1" or "2019 started as a good year, but 2019-H2 will bring challenges".